sql/core/src/test/scala/org/apache/spark/sql/test/DataFrameReaderWriterSuite.scala

```diff
@@ -224,17 +224,6 @@ class DataFrameReaderWriterSuite extends QueryTest with SharedSparkSession with
     assert(LastOptions.parameters("opt3") == "3")
   }
 
-  test("SPARK-32364: path argument of load function should override all existing options") {
```

Comment:
This is no longer relevant, so I am removing it. But I am keeping the case insensitivity check in the new test.

Reply:
We can just call `load` without parameters, this test is to verify that the later option overwrites the earlier ones.

Reply:
OK. I put this back and renamed the test. Thanks.